ui/apps/portal/src/components/Chat/Chat.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useMemo, useCallback } from "react";
+import { AiChat, useAiChatApi, useAsStreamAdapter } from "@nlux/react";
+import { Drawer } from "@mui/material";
+import "@nlux/themes/nova.css";
+
+import { createSend, noOpSend } from "./Send";
+import { useConversationHistory } from "../../contexts";
+import "./Chat.scss";
+
+export interface ChatProps {
+  open?: boolean;
+  onClose?: () => void;
+  datasetId?: string;
+  currentContent: () => any;
```

Comment:
in the use case, currentContent refers to the starboard notebook, but it can be any other context as well